framework/src/mesh/MooseMesh.C

```diff
@@ -3177,6 +3177,9 @@ MooseMesh::buildFaceInfo()
 const FaceInfo *
 MooseMesh::faceInfo(const Elem * elem, unsigned int side) const
 {
+  if (_face_info_dirty)
```

Comment:
This check is redundant with the one in the `buildFaceInfo` func.

Reply:
Ok removed!